README.md

```diff
@@ -1,99 +1,115 @@
-  'root' => '/path_to_drupal_root/',
-  'remote-host' => 'remote-host.princeton.edu',
-  'remote-user' => 'my-remote-user',
-  'ssh-options' => '-o PasswordAuthentication=no -i /path_to_my_ssh_alias',
-  'path-aliases' => array(
-    '%dump-dir' => '/path_to_remote_db_dump_directory/',
-    '%dump' => '/path_to_remote_db_dump_directory/recap_sql_sync_dump.sql',
-    '%sites' => '/path_to_drupal_root/sites/',
-    '%files' => 'sites/default/files',
-    ## You must have an executable drush 8 environment on the remote server
-    ## available at a path your remote user can executable
-    ## See instructions for install
-    '%drush-script' => '/path_to_drupal_root/vendor/bin/drush',
-  ),
-   'databases' =>
+    ```
+    $databases = array (
+      'default' =>
       array (
         'default' =>
         array (
-          'default' =>
-          array (
-            'driver' => 'mysql',
-            'username' => 'db_user',
-            'password' => 'db_passwd,
-            'port' => '',
-            'host' => 'localhost',
-            'database' => 'db_name',
-          ),
-       ),
-     ),
-  'command-specific' => array (
-    'sql-sync' => array (
-      'simulate' => '0',
-    ),
-    'rsync' => array (
-      'simulate' => '0',
-    ),
-  ),
-);
-```
+          'database' => 'drupal8',
+          'username' => 'drupal8',
+          'password' => 'drupal8',
+          'host' => 'database',
+          'port' => '3306',
+          'driver' => 'mysql',
+          'prefix' => '',
+        ),
+      ),
+    );
+    ```
+1. Add the following useful local development configuration to the end of `sites/default/settings.php`
+    ```
+    /* Overrides for the local environment */
+    $conf['securepages_enable'] = 0;
+    /* This should be set in your php.ini file */
+    ini_set('memory_limit', '1G');
+    /* Turn off all caching */
+    $conf['css_gzip_compression'] = FALSE;
+    $conf['js_gzip_compression'] = FALSE;
+    $conf['cache'] = 0;
+    $conf['block_cache'] = 0;
+    $conf['preprocess_css'] = 0;
+    $conf['preprocess_js'] = 0;
+    /* end cache settings */
+    /* Turn on theme debugging. Injects the path to every Template utilized in the HTML source. */
+    $conf['theme_debug'] = TRUE;
 
-### For database content
-```
-/path_to_local_drush_8_executable/drush @recap-dev sql-sync @recap-dev @recap-local
-```
+    /* Makes sure jquery is loaded on every page */
+    /* set to false in production */
+    $conf['javascript_always_use_jquery'] = TRUE;
 
-### For Files
-```
-/path_to_local_drush_8_executable/drush rsync @recap-dev:%files/ @recap-local:%files
-```
+    $settings['trusted_host_patterns'] = [
+      '^recap.lndo.site$',
+    ];
+    ```
+1. `mkdir .ssh` # excluded from version control
+1. `cp $HOME/.ssh/id_rsa .ssh/.`
+1. `cp $HOME/.ssh/id_rsa.pub .ssh/.` // key should be registered in princeton_ansible deploy role
+1. `lando start`
+1. `cp drush/recap-example.aliases.drushrc.php drush/recap.aliases.drushrc.php`
```

Comment:
I do not have a drush directory in this branch.

Reply:
I ignored the whole directory 🤦‍♂ I've added a commit to resolve this issue.